### PR TITLE
edit typos and grammar

### PR DESCRIPTION
**Pull request by Kiresorg**
Most are simple typo or grammar fixes. Only on `troubleshooting/main-entry-point-does-not-exist.md` did I take editorial license, adding an option for what to do if a dev chooses not to define a 'main' entry in package.json. I did this because it looked liked you intended to provide that info. 

I really like this project. Well done!

Let me know your thoughts.

### CODE REVIEW

File: docs/troubleshooting/main-entry-point-does-not-exist.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Either you can define a `main` entry that points to your entry point for example
 }
 ```
 
+Or, you can ensure the entrance point to your Express application is a `server.js` file in the root directory.
```
Comment (seekayel) [inline docs/troubleshooting/main-entry-point-does-not-exist.md:37]: Did you mean: `entry point`?

**Review by seekayel — COMMENTED** [review]
Awesome stuff here. One small possible change.

Also, we are double checking our validation logic for checking if the system can launch given the `package.json`. You may have uncovered a bug 👍

**Comment by Kiresorg** [comment]
Great catch. Yes - that’s keeps it consistent. I’ll make the edit in a few
minutes.

**Comment by Kiresorg** [comment]
Awesome! Glad to help. Let me know if there's any other way I can
contribute. I am an instructor and curriculum designer at Pluralsight; I
could make tutorials, or add to the docs?

Erik
ᐧ

**Review by seekayel — APPROVED** [review]
Awesome! Running checks, will merge.

Help with tutorials or starter/example repos sound great!

**Pull request merged**